src/test/java/core/basesyntax/stategyimpltest/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.stategyimpltest;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.BalanceOperationHandler;
+import core.basesyntax.strategy.OperationHandler;
+import java.util.Map;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class BalanceOperationHandlerTest {
+    private static Map<String, Integer> fruit;
+    private OperationHandler operationHandler;
+    private FruitTransaction fruitTransaction;
+
+    @BeforeEach
+    public void setUp() {
+        operationHandler = new BalanceOperationHandler();
+        fruit = Storage.FRUIT_MAPS;
+    }
+
+    @AfterAll
+    static void afterAll() {
+        fruit.clear();
+    }
+
```

Comment:
what about bad-path cases? e.g. not valid input, null-input and et cetera

tests should cover different scenarios